### ISSUE TITLE
[Master] Enable UapAot test variations on Windows. (VSTS)

### ISSUE THREAD

**Issue by StephenBonikowsky** [dotnet/wcf#2181]
Branch: MASTER

Currently IISHosted variation is enabled with roughly 30% of the UWP ILC tests run in VSTS failing with the following issue...

```
System.MissingMethodException : Method 'HttpBaseProtocolFilter.add_ServerCustomValidationRequested(TypedEventHandler<HttpBaseProtocolFilter, HttpServerCustomValidationRequestedEventArgs>)' was not included in compilation, but was referenced in HttpClientHandler.SetOperationStarted(). There may have been a missing assembly.

Stack Trace:
   E:\A\_work\499\s\corefx\src\System.Net.Http\src\uap\System\Net\HttpClientHandler.cs(584,0): at System.Net.Http.HttpClientHandler.<SendAsync>d__109.MoveNext()

   --- End of stack trace from previous location where exception was thrown ---
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs(61,0): at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(178,0): at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(147,0): at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(304,0): at System.Runtime.CompilerServices.ConfiguredTaskAwaitable$1<System.__Canon>.ConfiguredTaskAwaiter.GetResult()
   E:\A\_work\499\s\corefx\src\System.Net.Http\src\System\Net\Http\HttpClient.cs(489,0): at System.Net.Http.HttpClient.<FinishSendAsyncUnbuffered>d__59.MoveNext$catch$0()

   --- End of stack trace from previous location where exception was thrown ---
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs(61,0): at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(178,0): at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(147,0): at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
 f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(304,0): at System.Runtime.CompilerServices.ConfiguredTaskAwaitable$1<System.__Canon>.ConfiguredTaskAwaiter.GetResult()
     at System.ServiceModel.Channels.HttpChannelFactory$1<System.__Canon>.HttpClientRequestChannel.HttpClientChannelAsyncRequest.<SendRequestAsync>d__13.MoveNext()

   --- End of stack trace from previous location where exception was thrown ---
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs(61,0): at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(178,0): at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(147,0): at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      at System.ServiceModel.Channels.RequestChannel.<RequestAsync>d__33.MoveNext()

   --- End of stack trace from previous location where exception was thrown ---
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs(61,0): at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(178,0): at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(147,0): at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(304,0): at System.Runtime.CompilerServices.ConfiguredTaskAwaitable$1<System.__Canon>.ConfiguredTaskAwaiter.GetResult()
      at System.ServiceModel.Channels.RequestChannel.<RequestAsyncInternal>d__32.MoveNext()

   --- End of stack trace from previous location where exception was thrown ---
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\ExceptionServices\ExceptionDispatchInfo.cs(61,0): at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(178,0): at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\Runtime\CompilerServices\TaskAwaiter.cs(147,0): at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      at System.Runtime.TaskHelpers.WaitForCompletionNoSpin<System.__Canon>(Task$1<__Canon> task)
      at System.ServiceModel.Channels.RequestChannel.Request($Message message)
E:\A\_work\353\s\wcf\src\System.Private.ServiceModel\tests\Scenarios\Client\ChannelLayer\RequestReplyChannelShapeTests.4.0.0.cs(105,0): at RequestReplyChannelShapeTests.IRequestChannel_Http_CustomBinding()
      at _$ILCT$.$ILT$ReflectionDynamicInvoke$.InvokeRetV(Object thisPtr, IntPtr methodToCall, ArgSetupState argSetupState, Boolean targetIsThisCall)
      at System.InvokeUtils.CalliIntrinsics.Call(IntPtr dynamicInvokeHelperMethod, Object thisPtrForDynamicInvokeHelperMethod, Object thisPtr, IntPtr methodToCall, ArgSetupState argSetupState)
   f:\dd\ndp\fxcore\CoreRT\src\System.Private.CoreLib\src\System\InvokeUtils.cs(400,0): at System.InvokeUtils.CallDynamicInvokeMethod(Object thisPtr, IntPtr methodToCall, Object thisPtrDynamicInvokeMethod, IntPtr dynamicInvokeHelperMethod, IntPtr dynamicInvokeHelperGenericDictionary, Object targetMethodOrDelegate, Object[] parameters, BinderBundle binderBundle, Boolean invokeMethodHelperIsThisCall, Boolean methodToCallIsThisCall)
```

**Comment by StephenBonikowsky** [comment]
Moving the build and test infrastructure to Arcade.
Will fix/resolve this issue as part of wcf #3422

**Comment by StephenBonikowsky** [comment]
As of right now not doing any more work related to UAP.
